routes/ppe.js

```diff
@@ -0,0 +1,202 @@
+var express = require('express');
+var router = express.Router();
+var models = require('../models');
+const { upload, validate, makeURI } = require('./utils');
+const webpush = require('web-push');
+const redis = require("redis");
+const client = redis.createClient(
+  process.env.REDIS_PORT || '6379',
+  process.env.REDIS_HOST || '127.0.0.1',
+  {
+    'auth_pass': process.env.REDIS_KEY,
+    'return_buffers': true
+  });
+const searchRadius = 10; //km
+const { check } = require("express-validator");
+
+client.on("error", function (error) {
+  console.error(error);
+});
+client.on("connect", function (res) {
+  console.log("connected to redis instance");
+});
+
+
+/*********************************************
+ ****************** Getters ******************
+ *********************************************/
+
+// View ppe on map
+router.get('/map', function (req, res, next) {
+  res.render('ppe-map', { lat: req.query.lat || 22, lng: req.query.lng || 84, zoom: req.query.zoom || 4.5 });
+});
+// View ppe as list
+router.get('/list', async function (req, res, next) {
+  try {
+    const availabilities = await models.Availability.findAll({ include: models.PPEType });
+    const requirements = await models.Requirement.findAll({ include: models.PPEType });
+    const manufacturings = await models.Manufacturing.findAll({ include: models.PPEType });
```

Comment:
Even though I like the async-await model, I guess this would fire 3 DB calls in series, shouldn't we have 3 promises and do a promise.all() so that they are done in parallel?

Reply:
good idea